sources/nodeUtils.ts

```diff
@@ -16,7 +16,7 @@ function getEndOfLine(content: string) {
 }
 
 export function normalizeLineEndings(originalContent: string, newContent: string) {
-  return newContent.replace(/\r?\n/g, getEndOfLine(originalContent));
+  return newContent.replaceAll(`{}`, getEndOfLine(originalContent));
```

Comment:
Why did you make this change?

Reply:
Have a look at the URL: https://github.com/nodejs/corepack/blob/b1a8d923577c0a544cb264d1b7b047f2fc3e11af/config.json#L60

Reply:
I think that's a search and replace that was done by mistake, probably related to this suggestion https://github.com/nodejs/corepack/pull/333#discussion_r1416099647
```suggestion
  return newContent.replace(/\r?\n/g, getEndOfLine(originalContent));
```

Reply:
Maybe we should change the name of the function. Since it doesn't normalize line endings right now